browser/src/control/Control.Toolbar.js

```diff
@@ -100,6 +100,30 @@ function onClick(e, id, item) {
 			map.toggleCommandState(getUNOCommand(item.uno));
 		}
 	}
+	else if (item.id === 'print-active-sheet') {
+		map.sendUnoCommand('.uno:.uno:DeletePrintArea');
```

Comment:
duplicated code

Reply:
please use `this._map.dispatch(command)` and put action implementation in: browser/src/control/Toolbar.js  in `dispatch` method

Reply:
okay got it